test/loadTest/orion_updates_ngsiv2.jmx

```diff
@@ -0,0 +1,719 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I would suggest to add two .jmx with the following names:

* orion_updates_ngsiv2_with_hw_metrics.jmx
* orion_updates_ngsiv2_without_hw_metrics.jmx

Both are the same .jmx, except that one is prepared for hw metric and the other isn't.

*